indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -182,10 +240,15 @@ export default function SearchApp() {
             onPageChange={setPage}
             loading={results.loading}
           />
-        ) : (
-          <NoResults query={q} />
         )}
       </Grid.Column>
     </Grid>
   );
 }
+
+SearchApp.propTypes = {
+  category: PropTypes.shape({
+    id: PropTypes.number,
+    title: PropTypes.string,
+  }).isRequired,
```

Comment:
Wouldn't it make more sense to pass a `null` category when not filtering by category (and make the attributes inside required in case one is passed?)

Reply:
Good point, I started with only the ID, and passing a null ID was being translated into an empty string in the backend db query.

Now that it's an object it makes more sense to make it `null`.